### PR TITLE
Add e2e test that creates a community

### LINKED ISSUE #6445: Automation: Create Communities
### Description

Create an acceptance test that create a community

### PR DESCRIPTION
**Pull request by jrainville**
Fixes #6445

### CODE REVIEW

File: test/ui-test/src/screens/StatusMainScreen.py [hunk 1]
```diff
@@ -38,7 +38,8 @@ def joinChatRoom(self, room: str):
         type(ChatNamePopUp.INPUT_ROOM_TOPIC_TEXT.value, room)
         click_obj_by_name(ChatNamePopUp.START_CHAT.value)
         
-    
+    def goToCommunityPortal(self):
```
Comment (alaibe) [inline test/ui-test/src/screens/StatusMainScreen.py:41]: nit, keep same naming/convention: `open_community_portal`

**Review by alaibe — APPROVED** [review]
good job!!

File: test/ui-test/testSuites/suite_status/shared/scripts/bdd_hooks.py [hunk 2]
```diff
@@ -13,6 +13,7 @@
 @OnScenarioStart
 def hook(context):
     erase_directory("../../../../../Status/data")
+    erase_directory("../../../../../Status/qt")
```
Comment (noeliaSD) [inline test/ui-test/testSuites/suite_status/shared/scripts/bdd_hooks.py:16]: Why is it needed?

File: test/ui-test/testSuites/suite_status/tst_createCommunity/test.feature [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+    As a user I want to create a community and chat
+
+    The following scenarios cover basic flows of creating a community
+
+    Background:
+        Given A first time user lands on the status desktop and generates new key
+        When user signs up with username tester123 and password TesTEr16843/!@00
+        Then the user lands on the signed in app
+        Then the user opens the community portal section
```
Comment (noeliaSD) [inline test/ui-test/testSuites/suite_status/tst_createCommunity/test.feature:22]: Maybe we can use the same syntax than previous "lands on" when opening a new "page" and also, apart from clicking the button, verify that the new view is there (added related comment in another point)

File: test/ui-test/src/screens/StatusCommunityPortalScreen.py [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+class CreateCommunityPopup(Enum):
+    SCROLL_BAR: str = "mainWallet_Add_Account_Popup_Main"
+    COMMUNITY_NAME_INPUT: str = "createCommunityNameInput_TextEdit"
+    COMMUNITY_DESCRIPTION_INPUT: str = "createCommunityDescriptionInput_TextEdit"
+    NEXT_SCREEN_BUTTON: str = "createCommunityNextBtn_StatusButton"
+    COMMUNITY_INTRO_MESSAGE_INPUT: str = "createCommunityIntroMessageInput_TextEdit"
+    COMMUNITY_OUTRO_MESSAGE_INPUT: str = "createCommunityOutroMessageInput_TextEdit"
+    DO_CREATE_COMMUNITY_BUTTON: str = "createCommunityFinalBtn_StatusButton"
+
+class StatusCommunityPortalScreen:
```
Comment (noeliaSD) [inline test/ui-test/src/screens/StatusCommunityPortalScreen.py:21]: Maybe here we can add a `verify_screen` method.

File: test/ui-test/src/screens/StatusMainScreen.py [hunk 5]
```diff
@@ -38,7 +38,8 @@ def joinChatRoom(self, room: str):
...
+        click_obj_by_name(MainScreenComponents.COMMUNITY_PORTAL_ICON.value)
```
Comment (noeliaSD) [inline test/ui-test/src/screens/StatusMainScreen.py:42]: Maybe here we can verify the new screen is displayed.

File: test/ui-test/testSuites/suite_status/shared/steps/communitySteps.py [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+    _statusMainScreen.goToCommunityPortal()
+
+@When("the user creates a community named |any|, with description |any|, intro |any| and outro |any|")
+def step(context, community_name, community_description, community_intro, community_outro):
+    _statusCommunitityPortal.createCommunity(community_name, community_description, community_intro, community_outro)
+
+
+@Then("the user lands on the community")
+def step(context): 
+    StatusCommunityScreen()
```
Comment (noeliaSD) [inline test/ui-test/testSuites/suite_status/shared/steps/communitySteps.py:21]: Maybe we can extend the verification here checking that the introduced params, like the `name` and `description` are the ones that we expect. 

**NOTE:** To pass data accross steps you can use `context` object.

**Review by noeliaSD — CHANGES_REQUESTED** [review]
Nice first automation contribution!! :)
Just left some minor comments/thoughts!

**Comment by jrainville** [comment]
@noeliaSD thanks for the comments. I updated.
Note that it need this statusQ PR merged first: https://github.com/status-im/StatusQ/pull/789

File: test/ui-test/src/screens/StatusMainScreen.py (hunk 1 above)
Comment (noeliaSD) [reply test/ui-test/src/screens/StatusMainScreen.py:41]: > nit, keep same naming/convention: `open_community_portal`

Yes, please, update the method name ;)

Comment (noeliaSD) [reply test/ui-test/src/screens/StatusMainScreen.py:41]: > nit, keep same naming/convention: `open_community_portal`

Yes, please, could you update the method name?! ;)

Comment (jrainville) [reply test/ui-test/src/screens/StatusMainScreen.py]: updated

**Comment by jrainville** [comment]
StatusQ PR was merged. This is ready to be merged when the review is done

**Review by noeliaSD — APPROVED** [review]

**Pull request merged**